src/SonarQube.Bootstrapper/Program.cs

```diff
@@ -33,6 +34,11 @@ public static int Main(string[] args)
         {
             var logger = new ConsoleLogger(includeTimestamp: false);
             Utilities.LogAssemblyVersion(logger, Resources.AssemblyDescription);
+#if IS_NET_FRAMEWORK
+            logger.LogInfo("Executing the .NET Framework build of the Scanner for MSBuild");
```

Comment:
Done to help with support; if the user includes even a normal log we'll know which build was being used.

Reply:
Is the message compatible with the previous one? Not sure if we are not duplicating text... Probably it is worth to add another format argument to the resource message for the targeted framework.

Reply:
I thought it looked ok:

> SonarQube Scanner for MSBuild 4.1
> Executing the .NET Framework build of the Scanner for MSBuild
> ...

I considered moving the _LogAssemblyVersion_ method to this class and changing it to a single method, but decided not to do that at the moment. _LogAssemblyVersion_ is only called from one other place that I think you've removed @Val (the PreProcessor.Program::Main method). If so, I'll change this logging once the rebase is done.

Reply:
?

Reply:
Sorry, @Val - wrong "Val" ;-)